orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls

```diff
@@ -43,6 +43,13 @@ include_access_config:
     - repl: 'include: "/etc/unbound/access.conf"'
     - unless: grep "/etc/unbound/access.conf" /etc/unbound/unbound.conf
 
+set_max_ttl:
+  file.replace:
+    - name: /etc/unbound/unbound.conf
+    - pattern: '# cache-max-ttl:.*'
+    - repl: 'cache-max-ttl: 30'
+    - unless: grep 'cache-max-ttl:\s30'  /etc/unbound/unbound.conf
```

Comment:
could you drop this? it's unnecessary

Reply:
what do you mean on "this" :D

Reply:
the line I added the comment to, the `unless` part

Reply:
4 line was selected 48-51